docs/CHANGELOG.rst

```diff
@@ -16,10 +16,15 @@ Added
 
 Changed
 -------
+- Change test files and modify test function for ``rnaseqc-qc`` process
+- Bump ``rnaseqc-qc`` process version
+- Rename ``filter_sense_rate`` to ``filter_rnaseqc_metrics`` in ``filter.py``
+  and add additional fields for filtering
 
 Fixed
 -----
-
+- Fix ``format_ucsc`` function in ``rnaseqc-qc`` process
```

Comment:
The change is substantial, not just a bug fix. This also reflects in the version where the middle number was changed. I would moved this to `Changed` section. The message could be "Changed the ordering of exons in `format_ucsc` function of `rnaseq-qc` process".